src/Symfony/Bundle/FrameworkBundle/FrameworkBundle.php

```diff
@@ -85,8 +85,10 @@ public function build(ContainerBuilder $container)
         $container->addCompilerPass(new AddCacheWarmerPass());
         $container->addCompilerPass(new AddCacheClearerPass());
         $container->addCompilerPass(new AddExpressionLanguageProvidersPass());
-        $container->addCompilerPass(new TranslationExtractorPass());
-        $container->addCompilerPass(new TranslationDumperPass());
+        if (class_exists('Symfony\Component\Translation\Translator')) {
```

Comment:
And for those as well. It was more about not even registering them if the translation component is not installed. But that's probably not really needed as this is for build only. I'm going to revert this change.